dali/kernels/imgproc/convolution/cutlass/threadblock/predicated_tile_access_iterator.h

```diff
@@ -585,10 +561,10 @@ class PredicatedTileAccessIterator<Shape_, Element_, layout::ColumnMajor, Advanc
   }
 
   /// Adds a pointer offset in units of Element
-  CUTLASS_HOST_DEVICE
-  void add_pointer_offset(LongIndex pointer_offset) {
-    iterator_.add_pointer_offset(pointer_offset);
-  }
+  // CUTLASS_HOST_DEVICE
```

Comment:
?

Reply:
I guess I could trim some unused code from here, but maybe it's easier to keep it closer to the source. That way it has a bigger chance of working with some other layouts.

Reply:
I will just revert this for now.